Data description/clean data (#6)
UBC-MDS/movielover_py: docs/proposal.md

# Proposal

## Motivation and Purpose

Our role: Data Analysts at a consulting firm

Target Audience: Movie enthusiasts 

Millions of movies are published on a variety of channels; however, filtering excellent movies could be time-consuming. If we could provide a list of popular movies based on customers' preferences and illustrate approximation of box offices from different dimensions, they will save a large amount of time on searching for high-quality movies. To solve this challenge, we propose building a data visualization app that allows movie enthusiasts to explore the top 10 popular movies, box offices and other relevant information. Our app can be served as a great exploratory tool for people who like visualization and friendly-user-interface to extract various data at the same time with minimum efforts to select the filters.

## Description of the Data

> This "movies" dataset is from the [Vega Datasets][1] collection. The dataset we are visualizing comes from the Vega dataset on Movies. The data was compiled by Vega to create a repository for example data to be used in their other packages as stated on their Vega Sources page.

Reviewer: ```suggestion
This "movies" dataset is from the [Vega Datasets][1] collection. The data was compiled by Vega to create a repository for example data to be used in other packages as stated on the Vega Sources page.

```

> This dataset contains 3,200 movies with 16 variables for each movie. Due to the time restriction of this project, we will only be visualizing 7 variables after 1980.

Reviewer: ```suggestion
This dataset contains 3,200 movies with 16 variables for each movie. By filtering out relevant features based on the purpose of our app, we decide to interactively visualize 7 variables of the movies that were created after 1980.

```

> Five of the variables we will be analysing describe the movie attributes including the title of the movie (Title), the duration of the movie, which genre the movie belongs to (Major_Genre), MPAA-rating and the year when the movie was originally released (Release_Date).

Reviewer: ```suggestion
There are five descriptive attributes of a movie including the title of a movie (Title), the duration of a movie (Running_Time_min), which genre the movie belongs to (Major_Genre), MPAA-rating and the year when the movie was originally released (Release_Date).

```

> The other two variables describe the outcomes of the movie through ratings (IMDB_Ratings) or US gross revenue. The dataset contains ratings from different sources, but we chose to visualize IMDB_Rating as it has 600 more values in the dataset than Rotten Tomatoes and relies on audience reviews. We will take the US gross revenue to summarize the revenue for each movie.

Reviewer: ```suggestion
The remaining two variables describe the popularity of a movie through ratings (IMDB_Ratings) and US gross revenue (US_Gross). The dataset contains ratings from two different sources, but we chose IMDB_Rating as it has fewer missing values than the Rotten Tomatoes rating. Additionally, we will use US Gross revenue as an approximation of the box office in North America.

```
